### PR TITLE
html change

### PR DESCRIPTION
**Pull request by Pozik7**
### HTML
Changed the name on the button to "Click on me"

### CSS
Added new class ".message"

### JS
Changed from alert to html type 
Changed var to const 

image

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
-
 var button = document.getElementById("myButton");
```
Comment (VictorHaimov-HHH) [inline script.js]: Avoid using `var`.
Instead use: `const `

**Review by VictorHaimov-HHH — CHANGES_REQUESTED** [review]
Great job!!!!!

a minor fix remaining and you're good to go! 

Please include brief description & picture in the PR.

**Comment by VictorHaimov-HHH** [comment]
Nice! 
No need to show me code in description.
Also, can you align the button to be centered with the description ?

**Comment by Pozik7** [comment]
> Nice! No need to show me code in description. Also, can you align the button to be centered with the description ?

Done 
image

**Review by VicDosya — APPROVED** [review]

**Pull request merged**